mysql/init.sql

```diff
@@ -6,6 +6,8 @@ CREATE TABLE customers (
                            customer_id BINARY(16),
                            customer_name VARCHAR(30),
                            customer_type VARCHAR(30),
+                           created_at datetime(6) not null ,
```

Comment:
```suggestion
                           created_at datetime(6) NOT NULL,
```
통일성을 맞춰서 예약어는 대문자로 쓰시는게 어떨까요?